overlord/snapstate/snapstate_remove_test.go

```diff
@@ -438,7 +438,7 @@ func (s *snapmgrTestSuite) TestParallelInstanceRemoveRunThrough(c *C) {
 		{
 			op:             "remove-snap-data-dir",
 			name:           "some-snap_instance",
-			path:           filepath.Join(dirs.SnapDataDir, "some-snap"),
+			path:           filepath.Join(dirs.SnapDataDir, "some-snap_instance"),
```

Comment:
why this change?

Reply:
It lines up with this change here: https://github.com/snapcore/snapd/pull/14092/files#r1647912200.

It doesn't have to happen, but I noticed that it was inconsistent when writing the new tests here.